nextcord/guild.py

```diff
@@ -3457,3 +3461,171 @@ async def register_application_commands(self, *commands: BaseApplicationCommand)
+        ...
+
+    @overload
+    async def create_automod_rule(
+        self,
+        *,
+        name: str,
+        event_type: EventType = EventType.message_send,
+        trigger_type: TriggerType,
+        presets: Optional[KeywordPresetType] = MISSING,
+        notify_channel: Optional[GuildChannel] = MISSING,
+        enabled: Optional[bool] = True,
+        exempt_roles: Optional[List[Role]] = MISSING,
+        exempt_channels: Optional[List[GuildChannel]] = MISSING,
+    ) -> AutoModerationRule:
+        ...
+
+    @overload
+    async def create_automod_rule(
+        self,
+        *,
+        name: str,
+        event_type: EventType = EventType.message_send,
+        trigger_type: TriggerType,
+        presets: Optional[KeywordPresetType] = MISSING,
+        notify_channel: Optional[GuildChannel] = MISSING,
+        enabled: Optional[bool] = True,
+        exempt_channels: Optional[List[GuildChannel]] = MISSING,
+    ) -> AutoModerationRule:
+        ...
+
+    async def create_automod_rule(
+        self,
+        *,
+        name: str,
+        event_type: EventType = EventType.message_send,
+        trigger_type: TriggerType,
+        keyword_filters: Optional[List[str]] = MISSING,
+        presets: Optional[KeywordPresetType] = MISSING,
+        notify_channel: Optional[GuildChannel] = MISSING,
+        timeout_seconds: Optional[int] = MISSING,
+        enabled: Optional[bool] = True,
+        exempt_roles: Optional[List[Role]] = MISSING,
+        exempt_channels: Optional[List[GuildChannel]] = MISSING,
+    ) -> AutoModerationRule:
+        """
+        Create an auto moderation rule.
+
+        You must have the :attr:`~Permissions.manage_guild` to be able to do this.
+        Parameters
+        ----------
+        name: :class:`str`
+            The name of this auto moderation rule.
+        event_type: :class:`EventType`
+            The event type that this rule should start finding. Default to :attr:`EventType.message_send`.
+        trigger_type: :class:`TriggerType`.
+            The trigger type of this rule.
+        keyword_filters: List[:class:`str`]
+            The words that this rule should filter.
+        presets: List[:class:`KeywordPresetType`]
+            The preset keywords that this rule should filter.
+        notify_channel: Optional[:class:`abc.GuildChannel`]
+            The channel that Discord will send an alert to this channel when the rule is triggered.
+            Either this or `timeout_seconds` must be provided.
+        timeout_seconds: Optional[:class:`int`]
+            The seconds to timeout the user that triggered this rule.
+            Either this or `notify_channel` must be provided.
+        enabled: Optional[:class:`bool`]
+            Whether if this rule is enabled or not. Default is True.
+        exempt_roles: Optional[List[:class:`Role`]]
+            The roles that should not be affected by this rule.
+        exempt_channels: Optional[List[:class:`abc.GuildChannel`]]
```

Comment:
see above suggestion
```suggestion
        notify_channel: :class:`abc.GuildChannel`
            The channel that Discord will send an alert to this channel when the rule is triggered.
            Either this or `timeout_seconds` must be provided.
        timeout_seconds: :class:`int`
            The seconds to timeout the user that triggered this rule.
            Either this or `notify_channel` must be provided.
        enabled: :class:`bool`
            Whether if this rule is enabled or not. Default is True.
        exempt_roles: List[:class:`Role`]
            The roles that should not be affected by this rule.
        exempt_channels: List[:class:`abc.GuildChannel`]
```